### ISSUE TITLE
sub-sampling didn't work when use SubSamplingImage 

### ISSUE THREAD

**Issue by jiyewushen** [saket/telephoto#94]
version:0.12.1
```kotlin
Scaffold(modifier = Modifier.fillMaxSize()) { innerPadding ->
    val zoomableState = rememberZoomableState()
    val imageState = rememberSubSamplingImageState(
        zoomableState = zoomableState,
        imageSource = SubSamplingImageSource.asset("aaa.jpg")
    )

    SubSamplingImage(
        modifier = Modifier
            .fillMaxSize()
            .padding(innerPadding)
            .zoomable(zoomableState),
        state = imageState,
        contentDescription = "image",
    )
}
```
![test image](https://github.com/user-attachments/assets/e8f0a1a4-f850-467b-9c63-ca8b5b509eba)

[Screen_recording_20240723_212541.webm](https://github.com/user-attachments/assets/42f969f5-4c5e-4d82-a973-92fb847b7ea1)

**Comment by saket** [comment]
That is such a long image! 😄

**Comment by saket** [comment]
A fix is available in `0.13.0-SNAPSHOT`. Is that something you can use in the meantime? I'll make a stable release sometime soon.

**Comment by jiyewushen** [comment]
> Is that something you can use in the meantime?
Sorry for the late reply and thanks for the heads up.

**Comment by saket** [comment]
Fixed in version `0.13.0`: https://github.com/saket/telephoto/releases/tag/0.13.0
